WXstrip.py

```diff
@@ -0,0 +1,105 @@
+# encoding: utf-8
```

Comment:
So this is for a UI-based version of the wordclock? what's the difference to the old gtk-based approach? is this working cross-platform wise?

Reply:
i hope so. Can you test that on linux ? gtk is not working on macos
Since Wx is based on qt it should also work on linux